src/systemctl/systemctl-util.c

```diff
@@ -36,6 +36,9 @@ int acquire_bus(BusFocus focus, sd_bus **ret) {
         assert(focus < _BUS_FOCUS_MAX);
         assert(ret);
 
+        if (!IN_SET(arg_runtime_scope, RUNTIME_SCOPE_SYSTEM, RUNTIME_SCOPE_USER))
+                return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "--global is not supported for the operation.");
```

Comment:
This function calls either `bus_connect_transport()` or `bus_connect_transport_systemd()`,
so I had a look how each of them handles `RUNTIME_SCOPE_GLOBAL`.

`bus_connect_transport()`:
* `arg_transport == BUS_TRANSPORT_REMOTE`: returns -EOPNOTSUPP,
* `arg_transport != BUS_TRANSPORT_REMOTE`: raises assertion.

`bus_connect_transport_systemd()`:
* `arg_transport == BUS_TRANSPORT_LOCAL`: raises assertion,
* `arg_transport != BUS_TRANSPORT_LOCAL`: returns -EOPNOTSUPP.

From this I conclude that the change is technically correct.

I'd suggest a little change of the wording: "--global is not supported for _this_ operation."
Maybe `EOPNOTSUPP` would be a bit more appropriate error code, but I'm not sure.

Reply:
Thank you. The suggestion is applied.